src/vivarium_public_health/results/observer.py

```diff
@@ -0,0 +1,34 @@
+from vivarium.framework.engine import Builder
+from vivarium.framework.results import Observer
+
+
+class HealthObserver(Observer):
```

Comment:
Nit: I think `PublicHealthObserver` is better, since this is vivarium *public* health.

Reply:
Also, I don't think these changes were supposed to be in this PR.